frontend/src/app/core/global_search/input/global-search-input.component.html

```diff
@@ -60,17 +61,17 @@
             </span>
 
             <div class="global-search--wp-content">
-              <span [textContent]="item.project" 
-              class="global-search--wp-project">
+              <span [textContent]="item.project"
+                    class="global-search--wp-project">
               </span>
 
               <a class="global-search--wp-id"
-                [ngClass]="uiStateLinkClass">
```

Comment:
this is a lin kinside of a link, pretty sure it can just be a span. I think this is invalid html